X/xtl/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase, Pkg
+
+name = "xtl"
+version = v"0.7.7"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/xtensor-stack/xtl", "a7c1c5444dfc57f76620391af4c94785ff82c8d6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/xtl
+cmake -B build -G Ninja \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release
+cmake --build build --parallel ${nproc}
+cmake --install build
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [AnyPlatform()]
+
+# The products that we will ensure are always built
+products = Product[
+    # This is a header-only library without any products
```

Comment:
Could list some header files as `FileProduct`s?  That has the benefit of checking that the files actually exist.

Reply:
Done.